tests/base/index_set_31.cc

```diff
@@ -0,0 +1,71 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 - 2018 by the deal.II authors
```

Comment:
```suggestion
// Copyright (C) 2020 by the deal.II authors
```

Reply:
sorry about that, it is my first pull request ;)

Reply:
No worries. This is really good for a first pull request. You even included a test. :slightly_smiling_face:

Reply:
Indeed -- we forget this about 75% of the time for new tests. No need to worry, and as @masterleinad says, we're excited to have you among the authors of deal.II!